github/util_v4_branch_protection.go

```diff
@@ -210,6 +231,20 @@ func setDismissalActorIDs(actors []DismissalActorTypes) []string {
 	return pushActors
 }
 
+func setBypassPullRequestActorIDs(actors []BypassPullRequestActorTypes) []string {
+	bypassActors := make([]string, 0, len(actors))
+	for _, a := range actors {
+		if a.Actor.Team != (Actor{}) {
+			bypassActors = append(bypassActors, a.Actor.Team.ID.(string))
+		}
+		if a.Actor.User != (Actor{}) {
+			bypassActors = append(bypassActors, a.Actor.User.ID.(string))
+		}
```

Comment:
Do you mind explaining this logic to me a little more?

Reply:
The logic is the same as in the case of `setPushActorIDs` bellow.